src/Uno.UI.FluentTheme.v1/FluentMerge.targets

```diff
@@ -35,7 +35,7 @@
 	<Message Text="Theme resources XAML file was generated" />
   </Target>
 
-  <PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0' and '$(UnoRuntimeIdentifier)'=='Reference'">
```

Comment:
This is a potential behavior change. `$(TargetFramework)` is supposed to never be `netstandard2.0`.